src/codecs/decoders.ts

```diff
@@ -1,14 +1,16 @@
 import { builtinDecode, sniffMimeType, canDecodeImage } from '../lib/util';
 import Processor from './processor';
 import webpDataUrl from 'url-loader!./tiny.webp';
+import avifDataUrl from 'url-loader!./tiny.avif';
```

Comment:
This is over 300 bytes 😢 

I just took a 1x1 black pixel and played with settings until it was as small as possible. There might be better ways.

Reply:
I remember @kornelski mentioning on Twitter that Chrome currently requires too much metadata on AVIF images, whereas Firefox is more lax. Probably the case here as well - we could remove some more metadata, but it wouldn't render in Chrome?

Reply:
Yeah there's https://bugs.chromium.org/p/chromium/issues/detail?id=1120973. But yeah, rendering in Chrome is pretty important here 😄

Reply:
300 bytes sounds about right for AVIF. The header is like Oscars speech: "shout out to MPEG-4, and my cousins from the HEIF spec. And big thanks to ISO BMFF, and a chunk of love for MIAF spec, and let's not forget video manipulation tools and their backwards compatibility layers".

Alpha channel is supported by adding "auxiliary image" attachment with its own chunk that contains an index of locations where it has bytes in the file, and duplicate metadata that re-states its an AV1 image, and a chunk that copies AV1 encoding parameters in case you want to know them without looking 200 bytes further in the file, and a copy of image size just to be clear. And then you need to add auxiliary image reference chunk that says auxiliary image is related to the primary image. And then add a properties chunk with properties container and properties association chunks that give the auxiliary image an auxiliary property that contains, and I'm not kidding, *a URN*, equal to "urn:mpeg:mpegB:cicp:systems:auxiliary:alpha".

Other codes have a bit in header that says alpha=1.

Reply:
@jakearchibald Cross-linking a potential alternative here: https://twitter.com/RReverser/status/1298346612067098627

Reply:
Btw, have you tried an empty (0x0) image instead? I imagine it won't help too much if the sample is dominated by metadata, but maybe it will cut size down at least a bit?

Reply:
@kornelski does that mean it can't render an image with alpha progressively? Since required data is stuffed at the end?

Reply:
Ah, I see you're already pushing on this https://github.com/AOMediaCodec/libavif/issues/287